ui/tutorial-renderer/build.gradle.kts

```diff
@@ -25,6 +26,7 @@ dependencies {
     implementation(libs.androidx.constraintlayout)
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.viewpager2)
+    implementation(libs.androidx.compose.material3)
```

Comment:
just a personal preference, but I try to alpha sort dependencies within the different dependency sections. This allows quicker digestion when searching to see if a dependency has already been added or needs to be added.

The rough list of dependency sections are:
1. other GodTools sub projects
2. Any Jetpack (AndroidX) dependencies
3. Any Firebase dependencies
4. Any GTO Support dependencies (this is an internal support library with a bunch of commonly used code in our android apps)
5. Other 3rd party dependencies
6. KAPT dependencies (These dependencies transform/generate code for us)
7. Test dependencies
8. Test KAPT dependencies